paper/paper.md

```diff
@@ -0,0 +1,92 @@
+    affiliation: 1
+  - name: Christina Ertural
+    orcid: 0000-0002-7696-5824
+    affiliation: 1
+  - name: Adam J. Jackson
+    orcid: 0000-0001-5272-6530
+    affiliation: 3
+  - name: Janine George
+    orcid: 0000-0001-8907-0336
+    affiliation: "1, 2"
+affiliations:
+ - name: Federal Institute for Materials Research and Testing, Department Materials Chemistry, Berlin, 12205, Germany
+   index: 1
+ - name: Friedrich Schiller University Jena, Institute of Condensed Matter Theory and Solid-State Optics, Jena, 07743, Germany
+   index: 2
+ - name: Science and Technology Facilities Council, Didcot, Oxfordshire, GB
+   index: 3
+date: August 2023
+bibliography: paper.bib
+
+---
+# Summary
+_Lobsterpy_ is a Python package developed to systematically analyze, 
+describe, and visualize LOBSTER computations results. Alongside its python 
+interface, it also provides an easy-to-use command line interface (CLI) that runs automatic analysis of 
+the computations and generates a summary of results and publication-ready 
+figures. Since its first release, its capabilities have been extended significantly. 
+Unlike earlier versions which could only automatically analyze Crystal Orbital Hamilton 
+Populations (COHPs)[@dronskowski1993crystal], Lobsterpy can now also 
+analyze crystal orbital overlap populations (COOP)[@hughbanks1983chains] and Crystal orbital 
+bond index (COBI)[@müller2021crystal] to extract summarized bonding information 
+comprising of electronic-structure based coordination environments, bond strengths, 
+most relevant bonds, and their corresponding bonding and anti-bonding contributions. 
+Furthermore, one can now also extract the most relevant orbital interaction information. 
+Additionally, featurize and structure graphs modules provide a pathway to generate features to be used further for machine learning studies. 
+The features section comprehensively overviews the functionalities of this package. 
+
+_Lobsterpy_ was used to produce the results in [@ngo2023dft, @naik2023quantumchemical]
+
+# Statement of need
+Although the idea of "chemical bonding" might seem perplexing from a 
+physical standpoint, it has been employed several times to explain 
+various chemical phenomena and material properties.[@das2023strong, @ertural2022first,
+@hu2023mechanism] With the recent 
+advances in automation frameworks for high-throughput computational 
+investigations, bonding analysis for thousands of crystalline materials 
+could be performed with few lines of code.[@george2022automated] This 
+automation helps reduce the common mistakes inexperienced users make 
+while performing bonding analysis. However, it is essential to systematically 
+generate inputs and post-process the output files consistently to have 
+reliable and reproducible results. Furthermore, 
+having data from high-throughput calculations ready to utilize as inputs 
+would benefit data-driven material science research. _Lobsterpy_ fullfills 
+this missing link.
```

Comment:
Could you add an example here how an LobsterPy analysis looks like? Script and plot, maybe?

Reply:
Yes will do add that. I was planning to add atleast two nice figures in the file 😃

Reply:
Awesome and we should also describe orbital-wise analysis, the COBI options, featurizer, quality checks etc